packages/rancher-logging/package.yaml

```diff
@@ -1,6 +1,6 @@
 url: https://kubernetes-charts.banzaicloud.com/charts/logging-operator-3.9.0.tgz
 packageVersion: 00
-releaseCandidateVersion: 06
+releaseCandidateVersion: 07
```

Comment:
This will need to be `08` based on https://github.com/rancher/charts/pull/995#pullrequestreview-593715679
I believe that PR will be merged first since it has committer approval.

Reply:
Updated.